mlocal/frags/Makefile.stub

```diff
@@ -96,24 +100,34 @@ actions_LIST := $(SOURCEDIR)/etc/actions/exec $(SOURCEDIR)/etc/actions/run $(SOU
 actions_INSTALL := $(DESTDIR)$(SYSCONFDIR)/singularity/actions
 
 CLEANFILES += $(singularity) $(go_BIN) $(go_OBJ) $(bash_completion)
-INSTALLFILES := $(singularity_INSTALL) $(bash_completion_INSTALL)
+INSTALLFILES := $(singularity_INSTALL) $(bash_completion_INSTALL) $(config_INSTALL)
+ALL := $(go_BIN) $(bash_completion) $(config)
 
 ifeq ($(BUILD_RUNTIME),1)
 
-CLEANFILES += $(libruntime) $(starter) $(cni_plugins_EXECUTABLES)
-INSTALLFILES += $(starter_INSTALL) $(starter_suid_INSTALL) $(sessiondir) \
-	$(config_INSTALL) $(dist_bin_SCRIPTS_INSTALL) $(capability_JSON) $(syecl_config_INSTALL) \
-	$(actions_INSTALL) $(cni_plugins_INSTALL) $(cni_config_INSTALL) $(seccomp_profile_INSTALL) \
-	$(NVIDIA_liblist_INSTALL) $(cgroups_config_INSTALL)
+CLEANFILES += $(starter)
+INSTALLFILES += $(starter_INSTALL) $(sessiondir) $(dist_bin_SCRIPTS_INSTALL) \
+                $(capability_JSON) $(syecl_config_INSTALL) $(actions_INSTALL) $(seccomp_profile_INSTALL) \
+                $(NVIDIA_liblist_INSTALL) $(cgroups_config_INSTALL)
+ALL += $(starter)
+
+endif # build_runtime
 
-all: cscope collect $(libruntime) $(go_BIN) $(config) $(starter) $(bash_completion) cniplugins
+ifeq ($(WITH_NETWORK),1)
```

Comment:
I'm still questioning if this is the best approach. I don't think putting conditionals into our Makefile is necessarily a great idea. There is already a paradigm of `mconfig` conditionally adding contents to the final Makefile based on certain criteria. For example, `go_runtime_opts.mk` already exists to add specific options for when we are compiling with the runtime. Previously that file was being used to set go build tags for `runtime` build tag. I think it would be cleaner if we moved the code within these conditionals into their own file analogous to `go_runtime_opts.mk` and include those stubs in the final Makefile if appropriate.

For a naming scheme, I would suggest the following:

`Makefile_runtime.stub` - All rules for runtime
`Makefile_suid.stub` - All rules for including suid binary
`Makefile_network.stub` - All rules for CNI plugins

I think that this approach is more in line with the intention of `makeit`